test/e2e/set/scyllacluster/scyllamanager.go

```diff
@@ -267,82 +272,431 @@ var _ = g.Describe("Scylla Manager integration", func() {
 		o.Expect(tasks.TaskListItemSlice).To(o.BeEmpty())
 	})
 
-	g.It("should register cluster and sync backup tasks", func() {
+	provideGCSCredentials := func(ctx context.Context, sc *scyllav1.ScyllaCluster) *scyllav1.ScyllaCluster {
+		scCopy := sc.DeepCopy()
+
+		gcServiceAccountKey := f.GetGCSServiceAccountKey()
+		o.Expect(gcServiceAccountKey).NotTo(o.BeEmpty())
+
+		secret := &corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "gcs-service-account-key-",
+			},
+			Data: map[string][]byte{
+				"gcs-service-account.json": gcServiceAccountKey,
+			},
+		}
+
+		secret, err := f.KubeClient().CoreV1().Secrets(f.Namespace()).Create(ctx, secret, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		for i := range scCopy.Spec.Datacenter.Racks {
+			scCopy.Spec.Datacenter.Racks[i].Volumes = append(scCopy.Spec.Datacenter.Racks[i].Volumes, corev1.Volume{
+				Name: "gcs-service-account",
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{
+						SecretName: secret.Name,
+						Items: []corev1.KeyToPath{
+							{
+								Key:  "gcs-service-account.json",
+								Path: "gcs-service-account.json",
+							},
+						},
+					},
+				},
+			})
+			scCopy.Spec.Datacenter.Racks[i].AgentVolumeMounts = append(scCopy.Spec.Datacenter.Racks[i].AgentVolumeMounts, corev1.VolumeMount{
+				Name:      "gcs-service-account",
+				ReadOnly:  true,
+				MountPath: "/etc/scylla-manager-agent/gcs-service-account.json",
+				SubPath:   "gcs-service-account.json",
+			})
+		}
+
+		return scCopy
+	}
+
+	g.It("should register cluster, sync backup tasks and support manual restore procedure", func() {
```

Comment:
> This should be a different suite so we can distinguish running tests with and without object storage.

Do you mean that this should be an entirely separate suite, with a dedicated run, separate from the parallel suite? Or should the parallel suite have two versions, with and without storage?

Reply:
> probably a separate Describe so you can enforce that label on a set of tests that need objects storage

pretty sure you can add labels to `It`s as well, so we can pick and choose in suites

Reply:
added the label and an extra suite, the default parallel suite will require object storage

Reply:
> Do you mean that this should be an entirely separate suite, with a dedicated run, separate from the parallel suite? Or should the parallel suite have two versions, with and without storage?

yeah, if it would be a separate suite, I think we'd have to drop the conformance prefix (conformance needs backups working). Maybe just having a label that we can use with `--skip` would be enough for now and we'll see how it goes.

> pretty sure you can add labels to Its as well, so we can pick and choose in suites

It can, and Describe is the way to have that commonly enforce for every It in that block, so no single future test forgets that.

Reply:
> It can, and Describe is the way to have that commonly enforce for every It in that block, so no single future test forgets that.

but it also artificially groups tests into ones requiring object storage, which may not even test the same part of the system. Unless you mean a separate Describe block just for the manager integration tests requiring object storage

Reply:
> but it also artificially groups tests into ones requiring object storage, which may not even test the same part of the system.

Not all object storage tests would be in the single Describe. Just splitting the manager test into 2 Describes, say regular ones and extended that need object storage. You could even nest the Describes, but it may be too much indentation.

> should I remove the extra suite then?

yep, it wouldn't be called "conformance" and I think we are good with label skip to start with

Reply:
done